src/test/javascript/spec/component/shared/feedback-list.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { FeedbackItem, FeedbackItemType } from 'app/exercises/shared/result/result-detail.component';
```

Comment:
Not new tests, just more expanded form of them, as they were availble in `result.detail.component.spec.ts`